alica_engine/CMakeLists.txt

```diff
@@ -4,56 +4,101 @@ project(alica_engine)
 add_compile_options(-std=c++17)
 set(CMAKE_BUILD_TYPE Debug)
 
+# Set up utils to allow Colcon/catkin to find the packages
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+set(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/${CMAKE_PROJECT_NAME}/cmake) 
+
+
 ##Yaml
 find_package(yaml-cpp REQUIRED)
 
-##ROS/Catkin
-find_package(catkin QUIET)
-if (catkin_FOUND)
-  find_package(catkin REQUIRED COMPONENTS fsystem alica_common_config alica_solver_interface)
-else (catkin_FOUND)
-  set(catkin_LIBRARIES "fsystem" "alica_common_config" "alica_solver_interface")
-endif(catkin_FOUND)
-
-if(catkin_FOUND)
-  catkin_package(
-    INCLUDE_DIRS include
-    LIBRARIES alica_engine
-    CATKIN_DEPENDS fsystem alica_common_config alica_solver_interface
-  )
-endif(catkin_FOUND)
+# Imports
+find_package(fsystem REQUIRED)
+find_package(alica_common_config REQUIRED)
+find_package(alica_solver_interface REQUIRED)
+
 
 ###########
 ## Build ##
 ###########
 
-include_directories(include ${catkin_INCLUDE_DIRS} ${YAML_CPP_INCLUDE_DIR})
+include_directories(include 
+  ${alica_common_config_INCLUDE_DIRS}
+)
 
 file(GLOB_RECURSE alica_SOURCES "src/*.cpp")
```

Comment:
The common config is still needed since that package is just header files - it can't link it since there's nothing that gets compiled. If we merge this into alica_engine then that would address the include, but I think that should be in another PR